experimental/packages/opentelemetry-sdk-node/src/sdk.ts

```diff
@@ -121,11 +124,15 @@ export class NodeSDK {
     this._configuration = configuration;
 
     this._resource = configuration.resource ?? new Resource({});
-    this._resourceDetectors = configuration.resourceDetectors ?? [
-      envDetector,
-      processDetector,
-      hostDetector,
-    ];
+    let defaultDetectors: (Detector | DetectorSync)[] = [];
+    if (env.OTEL_NODE_RESOURCE_DETECTORS.length > 0) {
```

Comment:
We may be able to do:
```suggestion
    if (process.env.OTEL_NODE_RESOURCE_DETECTORS != null) {
```

This way we also don't need to add it to the env handling in `@opentelemetry/core` (which I hope we can remove in 2.0). The `getResourceDetectorsFromEnv()` code already does handle it via `process.env`, so nothing to change there :slightly_smiling_face: